weave-js/src/core/client/cachePolicy.ts

```diff
@@ -24,12 +24,33 @@ const getStringValFromNode = (
   return undefined;
 };
 
+const WANDB_COMMIT_HASH_LENGTH = 20;
+
+const isHexString = (s: string): boolean => {
+  return s.length % 2 === 0 && /^[0-9a-f]+$/i.test(s);
```

Comment:
I'd remove `s.length % 2 === 0` - the function name doesn't suggest it is doing that check, and it is covered by the exact length check in `likelyCommitHash` (could reverse order to do length check before regexp test)

Reply:
👍